### PR TITLE
Updated the MPD recipe to latest released version

### PR DESCRIPTION
**Pull request by trungnt2910**
I've updated MPD to the latest release version [0.21.19](http://www.musicpd.org/download/mpd/0.21/mpd-0.21.19.tar.xz).
The updated recipe builds and packages properly, also, the package can install.

[mpd_build.log](https://github.com/haikuports/haikuports/files/4082880/mpd_build.log)

### CODE REVIEW

**Comment by trungnt2910** [comment]
> Please temove the old patchset.

OK.

File: media-sound/mpd/patches/mpd-0.21.19.patchset [hunk 1]
```diff
@@ -1,15 +1,20 @@
-From b1fcd53d895256d41cf95945328c57877cb3c8c6 Mon Sep 17 00:00:00 2001
-From: =?UTF-8?q?Fran=C3=A7ois=20Revol?= <revol@free.fr>
-Date: Thu, 31 Jul 2014 23:09:29 +0200
-Subject: Haiku: add icon as resource definition file
+From 98b34984d4d8e463d006a3c052f519639c508260 Mon Sep 17 00:00:00 2001
+From: trungnt2910 <trungnt282910@gmail.com>
```
Comment (Begasus) [inline media-sound/mpd/patches/mpd-0.21.19.patchset]: There are some minor changes in the patchset as far as I can tell, could you keep the credits from the original creator of these?

Comment (trungnt2910) [reply media-sound/mpd/patches/mpd-0.21.19.patchset]: Actually, the original `patchset` works fine.
I'm just merging the two changes to the .rdef file
So should I keep the old one?

Comment (Begasus) [reply media-sound/mpd/patches/mpd-0.21.19.patchset]: Keep the old one, merge your changes up on the old patchset, that way the both of you will have/keep their credits for the changes :)

Comment (trungnt2910) [reply media-sound/mpd/patches/mpd-0.21.19.patchset]: I don't know if it can work, give me a while to test packaging with the old and renamed patchset.

Comment (trungnt2910) [reply media-sound/mpd/patches/mpd-0.21.19.patchset]: I've checked it, and the package builds properly. Thanks.

Comment (Begasus) [reply media-sound/mpd/patches/mpd-0.21.19.patchset]: Thanks!

**Comment by Begasus** [comment]
@extrowerk ... could you confirm if it's OK?

**Comment by extrowerk** [comment]
@Begasus : afternoon, i think.

**Comment by trungnt2910** [comment]
@extrowerk 
Please confirm my work, I think this is already afternoon in your time zone.
I'm waiting to claim another interesting task before the deadline...
Thanks.

**Comment by extrowerk** [comment]
Seems to be ok.

**Comment by trungnt2910** [comment]
Thanks.

**Comment by Begasus** [comment]
gcc2h package is broken (dependency for boost not correct):
https://build.haiku-os.org/buildmaster/master/x86_gcc2/logviewer.html?buildruns/3824/builds/34359.log

**Comment by trungnt2910** [comment]
> gcc2h package is broken (dependency for boost not correct):
> https://build.haiku-os.org/buildmaster/master/x86_gcc2/logviewer.html?buildruns/3824/builds/34359.log

So I guess that I should add ` >=1.58 ` to line `devel:libboost_system$secondaryArchSuffix` in the recipe?

**Comment by Begasus** [comment]
First delete remote and local branch
Then recreate local branch for a new PR, check the build on gcc2h if you can

**Pull request merged**